config/overlays/dev/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+# Adds namespace to all resources.
+namespace: gatekeeper-system
+
+bases:
+  - ../..
```

Comment:
This is causing issues for me `make deploy` using kustomize v3.0.3. 
```
Error: accumulating resources: accumulating resources from '../..': '/src/github.com/open-policy-agent/gatekeeper/config' must be a file
```

Reply:
Reworked the directory structure, for some reason this was broken in kustomize.

I filed a bug:

https://github.com/kubernetes-sigs/kustomize/issues/1391